src/natcap/invest/urban_cooling_model.py

```diff
@@ -57,7 +57,7 @@
             )
         },
         "ref_eto_raster_path": {
-            "name": "Reference Evapotranspiration Raster",
+            "name": "evapotranspiration",

```

Comment:
If this is dropping the "reference" part of the evapotranspiration, could we be sure to add it to the about section?